service/matching/taskListManager.go

```diff
@@ -459,7 +459,7 @@ func (c *taskListManagerImpl) DescribeTaskList(includeTaskListStatus bool) *type
 	response.TaskListStatus = &types.TaskListStatus{
 		ReadLevel:        c.taskAckManager.GetReadLevel(),
 		AckLevel:         c.taskAckManager.GetAckLevel(),
-		BacklogCountHint: c.taskAckManager.GetBacklogCount(),
+		BacklogCountHint: c.db.BacklogCount(),
```

Comment:
It's called by cadence-web and cadence-cli. And the value is only used to tell if there is backlog.

Reply:
based on your other comment this value will be potentially 5m old. If that's sufficient for now the change makes sense. otherwise we can maybe keep an accurate in-memory counter as tasks are added/removed from a tasklist in the future.

Reply:
Well, after some thought, I decide to not use the cached value. The api is not called frequently, and I don't think it will increase db load a lot.